### DISCUSSION TITLE
How to implement custom endpoints?

### DISCUSSION THREAD

**Discussion by ghost** [Azure/azure-mobile-apps#307] [Q&A] [answered]
Hi! I've been reading and following this post https://adrianhall.github.io/azure/2021/11/08/azure-mobile-apps-intro/.

I successfully ran the project! Quick question: how do I add some custom business logic to the endpoints? (i.e I would like to have a server side validation if the user owns the todo item to be updated)

**Comment by adrianhall** [comment, marked as answer]
Shockingly, that's the very next blog post in the sequence: https://adrianhall.github.io/azure/2021/11/10/complex-zumo-auth/
